src/services/navigator/helpers.test.ts

```diff
@@ -47,6 +48,67 @@ const screenDocSource =
  */
 const blankDoc = '<doc xmlns="https://hyperview.org/hyperview"></doc>';
 
+/**
+ * Test merge original document
+ */
+const mergeOriginalDoc = `
+<doc xmlns="https://hyperview.org/hyperview">
+  <navigator id="root-navigator" type="stack">
+    <nav-route id="tabs-route">
+      <navigator id="tabs-navigator" type="tab">
+        <nav-route id="live-shifts-route" href="{% url 'biz-app-hub' %}" initial="false"/>
```

Comment:
Could you remove any Django template tags in the mocked XML? (anything that starts with `{%`

Reply:
Done